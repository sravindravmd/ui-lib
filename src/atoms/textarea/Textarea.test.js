import React from 'react'
import { Textarea } from './'
import { render, cleanup } from '@testing-library/react'
import 'regenerator-runtime/runtime'

afterEach(cleanup)

const props = {
  className: 'multiline',
  label: 'Description',
  textareaProps: {
    limit: 100,
    rows: 5,
  },
}

describe('<Textarea/>', () => {
  it('renders Input component', () => {
    const { container } = render(<Textarea {...props} />)

    expect(container).toMatchSnapshot()
  })
})
