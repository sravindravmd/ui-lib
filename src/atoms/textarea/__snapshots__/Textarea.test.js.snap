// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Textarea/> renders Input component 1`] = `
<div>
  <div
    class="sc-bdfBwQ gqfcAk multiline"
  >
    <div
      class="sc-gsTCUz lauUiY"
    >
      <label
        class="textarea_parent"
      >
        <span
          class="textarea__label"
        >
          Description
        </span>
        <div
          class="sc-dlfnbm kYhlCE"
        >
          <textarea
            maxlength="100"
            rows="5"
          />
          <span
            class="limit"
          >
            <span>
              0
            </span>
            <span>
              /
              100
            </span>
          </span>
        </div>
      </label>
    </div>
  </div>
</div>
`;
